Automattic/wp-calypso
# Continue with GitHub: Add GitHub credentials to encrypted secrets file

Related to https://github.com/Automattic/wp-calypso/issues/87211

## Proposed Changes

Adds a `gitHubLoginUser` object to the secrets file.

## Testing Instructions

- Set E2E secrets key: PCYsg-vnR-p2 
- Decrypt the secrets 
- Verify that a `gitHubLoginUser` is present in `packages/calypso-e2e/src/secrets/decrypted-secrets.json`

## Pre-merge Checklist
